tests/functional/cylc-set/09-set-skip/flow.cylc

```diff
@@ -14,18 +14,18 @@
     [[graph]]
         R1 = """
             # Optional out not created by set --out skip
-            foo:no? => not_this_task?
+            foo:no? => not_this_task
 
             # set --out skip creates required, started, submitted
             # and succeeded (unless failed is set):
-            foo:yes => require_this_task
-            foo:submitted => submitted_emitted
-            foo:succeeded => succeeded_emitted
-            foo:started => skip_foo
+            foo:yes => foo_done
+            foo:submitted => foo_done
+            foo:succeeded => foo_done
```

Comment:
Consolidating these downstream tasks into a single one helps speed up the test by a few secs